tools/checked-c-convert/utils/headerConvertScript.py

```diff
@@ -0,0 +1,67 @@
+"""
+This program takes a list of .c files, e.g. in a convert.sh file,
+and converts any included headers for which checkedc has _checked 
+to the checked versions. The list of checked headers is found in
+the llvm/projects/checkedc-wrapper/checkedc/include directory.
+
+NOTE: This does not  any local / project-specific headers
+that have .checked.h versions from a previous run of checked-c-convert
+Those must still be updated manually.
+"""
+
+import os, re, sys
+import argparse
+
+# This assumes the file is inside the tools/checked-c-convert/utils
+# directory of a tools/checkedc-clang repo, which is inside an llvm repo
+# containing the checkedc-wrapper. Adjust if the headers are elsewhere.
+CHECKEDC_HEADER_DIR=os.path.abspath(
```

Comment:
I suggest adding an optional argument that sets this and using this value only if nothing is set.